code/__defines/fluids.dm

```diff
@@ -54,3 +54,9 @@
 		} \
 	} \
 	if(!flooded_a_neighbor) REMOVE_ACTIVE_FLUID_SOURCE(T);
+
+// We share overlays for all fluid turfs to sync icon animation.
+var/list/fluid_images = list()
+#define APPLY_FLUID_OVERLAY(img_state) \
+	if(!fluid_images[img_state]) fluid_images[img_state] = image('icons/effects/liquids.dmi',img_state); \
```

Comment:
Not really any benefit to using images if you're just leaving them as default anyways.

Reply:
Going to leave it explicit so that the base object can use a different icon without messing it up.